README.md

```diff
@@ -272,7 +272,8 @@ also of likely interest to founders and hiring managers:
 - [**Evaluating Equity Compensation**](#evaluating-equity-compensation): Whether you have equity now
   or will in the future, it is helpful to learn *how to think about* the value of equity and
   its tax burden.
-  We also cover whether you can [sell private stock](#can-you-sell-private-stock).
+  We also cover whether you can [sell private stock](#
```

Comment:
Broken line.

Reply:
I saw that in the diff and thought Flowmark must have done it and that I should leave it. Did I inadvertently break the line?

Reply:
Can you fix it, flowmark it again and verify? That's not right Markdown so either it's an error in editing or a bug in Flowmark (which you can report to me or report [here](https://github.com/jlevy/atom-flowmark/issues), where we're tracking issues).